Cargo.toml

```diff
@@ -100,6 +104,7 @@ service-registry = { version = "^1.1.0", path = "contracts/service-registry" }
 service-registry-api = { version = "^1.0.0", path = "packages/service-registry-api" }
 sha3 = { version = "0.10.8", default-features = false, features = [] }
 signature-verifier-api = { version = "^1.0.0", path = "packages/signature-verifier-api" }
+snarkvm-cosmwasm = { git = "https://github.com/eigerco/snarkVM", branch = "cosmwasm-4" }
```

Comment:
We need to update this before merging to point at our tag after the targeted branch is reviewed